blog/releases/announcing-knative-v0-17-release.md

```diff
@@ -0,0 +1,164 @@
+- For MTBroker, the Kubernetes event "BrokerReconciled" is no longer produced for clean runs of the FinalizeKind method. ([#3697](https://github.com/knative/eventing/pull/3697), [@n3wscott](https://github.com/n3wscott)
+- For Parallel, the Kubernetes event "ParallelReconciled" is no longer produced for clean runs of the ReconcileKind method. ([#3698](https://github.com/knative/eventing/pull/3698), [@n3wscott](https://github.com/n3wscott)
+- For PingSource, the Kubernetes event "PingSourceReconciled" is no longer produced for clean runs of the ReconcileKind method.
+- For Sequence, the Kubernetes event "SequenceReconciled" is no longer produced for clean runs of the ReconcileKind method. ([#3699](https://github.com/knative/eventing/pull/3699), [@n3wscott](https://github.com/n3wscott)
+- For Subscription, the Kubernetes event "SubscriptionReconciled" is no longer produced for clean runs of the ReconcileKind method. ([#3695](https://github.com/knative/eventing/pull/3695), [@n3wscott](https://github.com/n3wscott)
+- DeadLetterChannel was being dropped when converting between v1beta1<->v1 ([#3574](https://github.com/knative/eventing/pull/3574), [@vaikas](https://github.com/vaikas)
+- Not all the conditions were being properly converted between v1beta1<->v1. Basically only the Ready was.
+- Extend the terminationGracePeriod to fix issues shutting down the webhook. ([#3596](https://github.com/knative/eventing/pull/3596), [@mattmoor](https://github.com/mattmoor)
+- v1 and v1beta1 DeliverySpec.BackoffDelay accept ISO8601 duration ([#3619](https://github.com/knative/eventing/pull/3619), [@pierDipi](https://github.com/pierDipi)
+- PingSource does not lose events anymore when being shutdown close to the minute ([#3831](https://github.com/knative/eventing/pull/3831), [@lionelvillard](https://github.com/lionelvillard)
+
+</details>
+
+
+<details><summary>Other Changes</summary>
+
+- Add missing "leases" RBAC to controller and webhook to support leader election. ([#3562](https://github.com/knative/eventing/pull/3562), [@mattmoor](https://github.com/mattmoor)
+- Control plane components now specify anti-affinity so that replicas will not be colocated. ([#3795](https://github.com/knative/eventing/pull/3795), [@mattmoor](https://github.com/mattmoor)
+- The multi-tenant PingSource adapter consumes less resources. ([#3451](https://github.com/knative/eventing/pull/3451), [@lionelvillard](https://github.com/lionelvillard)
+- Reconcile eventing.{Broker,Trigger} using v1 api shape. Operate on dependent resources (Subscriptions, etc.) using their v1 shapes. ([#3587](https://github.com/knative/eventing/pull/3587), [@vaikas](https://github.com/vaikas)
+- When Trigger is reconciled, do not emit an event for it ([#3643](https://github.com/knative/eventing/pull/3643), [@vaikas](https://github.com/vaikas)
+
+</details>
+
+### Client
+
+TODO: check when is the kn cli 0.17 release cut
+
+### Operator
+
+TODO: check when is the operator 0.17 release cut
```

Comment:
Any idea who can confirm this, @knative/knative-release-leads ?

Reply:
This is now released, I have being working with the WGs over Slack to coordinate.

Reply:
https://github.com/knative/operator/releases/tag/v0.17.0